package/network/utils/iptables/Makefile

```diff
@@ -9,28 +9,26 @@ include $(TOPDIR)/rules.mk
 include $(INCLUDE_DIR)/kernel.mk
 
 PKG_NAME:=iptables
-PKG_VERSION:=1.4.21
-PKG_RELEASE:=2
+PKG_VERSION:=1.6.0
+PKG_RELEASE:=1
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.bz2
 PKG_SOURCE_URL:=http://www.netfilter.org/projects/iptables/files \
```

Comment:
Yes, I would however recommend you to place the www.netfilter.org as last site. It's not written anywhere but we're trying to be nice to upstream by using mirrors provided whenever possible as a nice gesture.